vector/src/main/java/im/vector/adapters/RoomInvitationViewHolder.java

```diff
@@ -43,6 +52,16 @@ void populateViews(final Context context, final MXSession session, final Room ro
                        final AbsAdapter.RoomInvitationListener invitationListener, final AbsAdapter.MoreRoomActionListener moreRoomActionListener) {
         super.populateViews(context, session, room, room.isDirectChatInvitation(), true, moreRoomActionListener);
 
+        if (null != room && room.isDirect()) {
```

Comment:
You should use "room.isDirectChatInvitation()" here instead of "room.isDirect()", this is more relevant for an invite. Even if I think both methods will return the same result

Reply:
Is that really necessary to check null != room whereas we already call "room.isDirectChatInvitation()" in super.populateViews call?
If this check is required, we should move it before super.populateViews